soroban-sdk/Cargo.toml

```diff
@@ -53,6 +53,7 @@ derive_arbitrary = { version = "~1.3.0" }
 proptest = "1.2.0"
 proptest-arbitrary-interop = "0.1.0"
 libfuzzer-sys = "0.4.7"
+expect-test = "1.4.1"
```

Comment:
👏🏻 This is pretty cool, love it's use here, seems very appropriate for these types of test expectations.